src/scripts/items/VitaminController.ts

```diff
@@ -0,0 +1,13 @@
+class VitaminController {
+    public static counter = 0;
+    public static vitaminmultipliers = ['×1', '×5', 'x10'];
```

Comment:
May be worth adding in a `Max` option,
Also camelCase the variable:
```suggestion
    public static vitaminMultipliers = ['×1', '×5', '×10', 'Max'];
```

Reply:
camelCase is done. Counter was because I overlooked removing it. Will adding max as an option automatically catch trying to apply more than available?

Reply:
Unfortunately not, it will needed to be coded in, as it will need to catch 5x and 10x not over applying proteins too